test/outdated/goldens/newer_versions.txt

```diff
@@ -0,0 +1,139 @@
+$ pub outdated --format=json
+Resolving...
+{
+  "packages": [
+    {
+      "package": "foo",
+      "current": "1.2.3",
+      "upgradable": "1.3.0",
+      "resolvable": "2.0.0",
+      "latest": "3.0.0"
+    },
+    {
+      "package": "builder",
+      "current": "1.2.3",
+      "upgradable": "1.3.0",
+      "resolvable": "2.0.0",
+      "latest": "2.0.0"
+    },
+    {
+      "package": "transitive",
+      "current": "1.2.3",
+      "upgradable": "1.3.0",
+      "resolvable": "1.3.0",
+      "latest": "2.0.0"
+    },
+    {
+      "package": "transitive2",
+      "current": null,
+      "upgradable": null,
+      "resolvable": "1.0.0",
+      "latest": "1.0.0"
+    },
+    {
+      "package": "transitive3",
+      "current": null,
+      "upgradable": null,
+      "resolvable": "1.0.0",
+      "latest": "1.0.0"
+    }
+  ]
+}
+
+$ pub outdated --format=no-color
+Resolving...
+Package      Current  Upgradable  Resolvable  Latest  
+dependencies 
+foo          *1.2.3   *1.3.0      *2.0.0      3.0.0   
+
+dev_dependencies
+builder      *1.2.3   *1.3.0      2.0.0       2.0.0   
+
+transitive dependencies
+transitive   *1.2.3   *1.3.0      *1.3.0      2.0.0   
+transitive2  -        -           1.0.0       1.0.0   
+transitive3  -        -           1.0.0       1.0.0   
+
+3 dependencies are currently not `pubspec.lock`'ed at the `Upgradable` version.
+They can be upgraded with `pub upgrade`.
+
+4 dependencies can be upgraded to the ‘Resolvable’ version by updating the
+constraints in `pubspec.yaml`.
```

Comment:
@kwalrath can you help with the wording of these sentences?

Reply:
How about something like this:

```
3 upgradable dependencies are locked (in pubspec.lock) to older versions.
To update these dependencies, use `pub upgrade`.

4 dependencies are constrained to versions that are older than a resolvable version.
To update these dependencies, edit pubspec.yaml.
```

Reply:
That's nice. Thanks!